### PR TITLE
Issue #4500: Wire up UI -> AddonManager

### PR DESCRIPTION
**Pull request by csadilek**
Next 🧩 . Wiring up UI and AddonManager.

uBlock can be installed/uninstalled with this.

### CODE REVIEW

File: samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt [hunk 1]
```diff
@@ -46,7 +49,43 @@ class InstalledAddonDetailsActivity : AppCompatActivity() {
...
+                        ).show()
+                    }
+                )
+            } else {
+                this.components.addonManager.enableAddon(
+                    addon,
+                    onSuccess = {
+                        Toast.makeText(
+                            this,
+                            "Successfully disabled ${addon.translatableName.translate()}",
```
Comment (Amejia481) [inline samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt]: nit: Enable

File: samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt [hunk 2]
```diff
@@ -46,7 +49,43 @@ class InstalledAddonDetailsActivity : AppCompatActivity() {
...
+                            Toast.LENGTH_SHORT
+                        ).show()
+                    },
+                    onError = {
+                        Toast.makeText(
+                            this,
+                            "Failed to disable ${addon.translatableName.translate()}",
```
Comment (Amejia481) [inline samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt]: nit: Enable

File: samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt (hunk 1 above)
Comment (csadilek) [reply samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt]: Updated!

File: samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt (hunk 2 above)
Comment (csadilek) [reply samples/browser/src/main/java/org/mozilla/samples/browser/addons/InstalledAddonDetailsActivity.kt]: 👍 updated.

**Comment by csadilek** [comment]
bors r=psymoon,Amejia481

**Comment by csadilek** [comment]
bors retry

**Pull request merged**